tiledb/sm/misc/stats.h

```diff
@@ -205,18 +205,18 @@ extern Statistics all_stats;
  * statement in the function. Note that a function can have multiple exit paths
  * (i.e. multiple returns), but you should still put this macro after the very
  * last statement in the function. */
-#define STATS_FUNC_OUT(f)                                     \
-  }                                                           \
-  ();                                                         \
-  if (stats::all_stats.enabled()) {                           \
-    auto __stats_end = std::chrono::steady_clock::now();      \
-    uint64_t __stats_dur_ns =                                 \
-        std::chrono::duration_cast<std::chrono::nanoseconds>( \
-            __stats_end - __stats_start)                      \
-            .count();                                         \
-    stats::all_stats.f##_total_ns += __stats_dur_ns;          \
-    stats::all_stats.f##_call_count++;                        \
-  }                                                           \
+#define STATS_FUNC_OUT(f)                                        \
```

Comment:
Can be reverted

Reply:
These actually can't be reverted currently -- the curl client does not use the `tiledb::sm` namespace, but it does use stats.

Reply:
Yeah everything in `rest/` is its its own `tiledb::rest` namespace.